_projects/git_task_list.md

```diff
@@ -0,0 +1,87 @@
+---
+collaborating_projects:
+  - coala
+  - meta-review
```

Comment:
no.  meta-review is being done by the other project.  Your project proposal did not propose any solutions related to meta-review.